web/src/compose_state.ts

```diff
@@ -194,6 +194,13 @@ export function has_message_content(): boolean {
     return message_content() !== "";
 }
 
+export const MINIMUM_MESSAGE_LENGTH_TO_SAVE_DRAFT = 2;
```

Comment:
I feel like ideally this would be in `drafts.js` but then there would be a circular import situation